docs/backends.md

```diff
@@ -114,3 +114,24 @@ These are the parameters for AWS:
 AVP_TYPE: awssecretsmanager
 AWS_REGION: Your AWS Region (Optional: defaults to us-east-2)
 ```
+
+### GCP Secret Manager
+
+#### GCP Authentication
```

Comment:
Sorry, one more small nitpick with this doc. Can you update the docs to look like this:

### GCP Secret Manager

##### GCP Authentication
Refer to the [Authentication Overview](https://cloud.google.com/docs/authentication) for Google Cloud APIs.

These are the parameters for GCP:
```
AVP_TYPE: gcpsecretmanager
```

##### Examples

###### Path Annotation
```yaml
kind: Secret
apiVersion: v1
metadata:
  name: test-secret
  annotations:
    avp.kubernetes.io/path: projects/12345678987/secrets/test-secret/versions/latest
type: Opaque
data:
  password: <test-secret>
```

###### Inline Path
```yaml
kind: Secret
apiVersion: v1
metadata:
  name: test-secret
type: Opaque
data:
  password: <path:projects/12345678987/secrets/test-secret/versions/latest#test-secret>
```

Thanks!